docs/fundamentals/code-analysis/configuration-files.md

```diff
@@ -76,14 +76,20 @@ is_global = true
 
 ### Naming
 
-Unlike EditorConfig files, which must be named `.editorconfig`, global config files do not need to have a specific name or extension. However, if you name these files as `.globalconfig` then they will be implicitly applied to all the C# and Visual Basic projects within the current folder, including subfolders. Otherwise, you must explicitly add the `GlobalAnalyzerConfigFiles` item to your MSBuild project file:
+Unlike EditorConfig files, which must be named `.editorconfig`, global config files do not need to have a specific name or extension. However, if you name these files as `.globalconfig`, then they are implicitly applied to all the C# and Visual Basic projects within the current folder, including subfolders. Otherwise, you must explicitly add the `GlobalAnalyzerConfigFiles` item to your MSBuild project file:
 
 ```xml
 <ItemGroup>
   <GlobalAnalyzerConfigFiles Include="<path_to_global_analyzer_config>" />
 </ItemGroup>
 ```
 
+Consider the following naming recommendations:
+
+- End users should name their global config files *.globalconfig*.
+- NuGet packages creators should name their global config files *<%Package_Name%>.globalconfig*.
```

Comment:
```suggestion
- NuGet package creators should name their global config files *<%Package_Name%>.globalconfig*.
```

Normally you'd have to escape `<` but it's working OK this way -- I don't know why.

Reply:
> Normally you'd have to escape < but it's working OK this way -- I don't know why.

It *may* not render correctly for all parsers.